pytket/phir/phirgen.py

```diff
@@ -375,9 +377,18 @@ def make_comment_text(cmd: tk.Command, op: tk.Op) -> str:
     """Converts a command + op to the PHIR comment spec."""
     comment = str(cmd)
     match op:
+        case tk.Conditional():
+            conditional_text = str(cmd)
+            cleaned = (
```

Comment:
I'm a bit puzzled as to why this is necessary. For a conditional WASM op, `str(cmd)` looks like `IF ([b[0]] == 1) THEN WASM a[0], a[1], _w[0];`; this is similar to how other conditional operations are represented, so why is special handling necessary?

Reply:
Here's the diff with the change and without:
```
 -     '//': "IF ([c1[0]] == 1) THEN WASM_function='add_one' args=['c0'] "
 -     "returns=['c0'];",
 +     '//': 'IF ([c1[0]] == 1) THEN WASM c0[0], c0[1], c0[2], c0[0], c0[1], c0[2], '
 +     '_w[0];',

```
ie, the condition is letting us rewrite WASM portion in a more user-friendly manner.

Reply:
Ah I see.